antho444/antho444/Ruby_Assignment.rb

```diff
@@ -0,0 +1,89 @@
+rooms = ["Bedroom", "Kitchen", "Blinds", "Livingroom" "Trash Can"]
```

Comment:
Blinds and Trash Can are not rooms - they could be objects in rooms if you want to include them in that way. While it's not always required to map code directly to real life, it can be confusing for the reader to see something that doesn't seem to be consistent. In this case, if you have a reason for treating these objects as rooms, it's a good idea to document it so the reader/user understands your thought process.

Reply:
Got it thanks for the feedback I appreciate it